### ISSUE TITLE
Starting emissions GDR and ECPC in 2021 have a small error

### ISSUE THREAD

**Issue by MarkMDekker** [imagepbl/effort-sharing#97]
This is likely due to incomplete data -> making the sum of all countries not exactly equal to the world. Maybe ignore for now?

For NLD this is 4% emissions change in 2021. That's not negligible.

**Comment by MarkMDekker** [comment]
@chantalwuer do you have an opinion on this? Maybe we can keep going with the data update, mention it in the update log and come back to this in January.

**Comment by MarkMDekker** [comment]
Specifically, this also means that if you look at the CO2 budgets export, there are (very) small differences in global budgets, if you subsequently add them up again.

**Comment by MarkMDekker** [comment]
Content-wise not a problem, but can be somewhat confusing

**Comment by chantalwuer** [comment]
@MarkMDekker I agree, we should move forward with the data update now but be transparent about this and address it as soon as possible in January.

**Comment by MarkMDekker** [comment]
OK. Let's note it down in the Zenodo v0.4 description and get back to it in January for the next (v0.5) version.

**Comment by MarkMDekker** [comment]
This is resolved for ECPC. Only applicable to GDR now.
